lib/deprecation-cop-status-bar-view.coffee

```diff
@@ -16,29 +16,31 @@ class DeprecationCopStatusBarView extends View
   toolTipDisposable: null
 
   initialize: ->
-    debouncedUpdateDeprecatedSelectorCount = _.debounce(@updateDeprecatedSelectorCount, 1000)
+    @debouncedUpdateDeprecatedSelectorCount = _.debounce(@updateDeprecatedSelectorCount, 1000)
 
     @subscriptions = new CompositeDisposable
     @subscriptions.add Grim.on 'updated', @update
-    @subscriptions.add atom.packages.onDidLoadPackage debouncedUpdateDeprecatedSelectorCount
-    @subscriptions.add atom.packages.onDidUnloadPackage debouncedUpdateDeprecatedSelectorCount
-    @subscriptions.add atom.packages.onDidActivatePackage debouncedUpdateDeprecatedSelectorCount
+    @subscriptions.add atom.packages.onDidLoadPackage @debouncedUpdateDeprecatedSelectorCount
+    @subscriptions.add atom.packages.onDidUnloadPackage @debouncedUpdateDeprecatedSelectorCount
+    @subscriptions.add atom.packages.onDidActivatePackage @debouncedUpdateDeprecatedSelectorCount
 
     @subscriptions.add atom.keymaps.onDidReloadKeymap (event) =>
-      @updateDeprecatedSelectorCount() if event.path is atom.keymaps.getUserKeymapPath()
+      @debouncedUpdateDeprecatedSelectorCount() if event.path is atom.keymaps.getUserKeymapPath()
```

Comment:
[Stylesheet related](https://github.com/atom/deprecation-cop/pull/58/files#diff-924d560c84e78032940fb0e2f39aa859R32) and `onDidReloadKeymap` callbacks are using debounce too.

I guess having the keymap callback debounced doesn't make that much sense as it doesn't seem to be activated during startup, so I can revert this back. *Let me know, senpai! :bow:*

Stylesheet-related callbacks are executed after window is loaded so it it not affecting Timecop numbers, but it is still saving several tens of miliseconds.